### ISSUE TITLE
The optimizer settings in config.py is correct ? it seems like  ...  emmm

### ISSUE THREAD

**Issue by zhishao** [gjy3035/NWPU-Crowd-Sample-Code#19]
__C.LR = 1e-4 # learning rate
__C.LR_DECAY = 1 # decay rate
__C.LR_DECAY_START = -1 # when training epoch is more than it, the learning rate will be begin to decay
__C.NUM_EPOCH_LR_DECAY = 1 # decay frequency
__C.MAX_EPOCH = 2000

the params are wrong ?
